.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: npm
```

Comment:
Can we group them? E.g. https://github.com/software-mansion/cairo-profiler/blob/main/.github/dependabot.yml, so it is all in one PR

Reply:
yes! I forgot about this feature